README.md

```diff
@@ -13,41 +13,44 @@ Requirements
 - [Terraform](https://www.terraform.io/downloads.html) 0.10+
 - [Go](https://golang.org/doc/install) 1.12 (to build the provider plugin)
 
-Building the Provider
+Developing the Provider
 ---------------------
 
-Clone repository to: `$GOPATH/src/github.com/terraform-providers/terraform-provider-aws`
+If you wish to work on the provider, you'll first need [Go](http://www.golang.org) installed on your machine (please check the [requirements](https://github.com/terraform-providers/terraform-provider-aws#requirements) before proceeding).
+
+*Note:* This project uses [Go Modules](https://blog.golang.org/using-go-modules) making it safe to work with it outside of your existing [GOPATH](http://golang.org/doc/code.html#GOPATH). The instructions that follow assume a directory in your home directory outside of the standard GOPATH (i.e `$HOME/development/terraform-providers/`). If you are using go1.11 please make sure that `GO111MODULE=on` is set.
+
+Clone repository to: `$HOME/development/terraform-providers/`
 
 ```sh
-$ mkdir -p $GOPATH/src/github.com/terraform-providers; cd $GOPATH/src/github.com/terraform-providers
+$ mkdir -p $HOME/development/terraform-providers/; cd $HOME/development/terraform-providers/
 $ git clone git@github.com:terraform-providers/terraform-provider-aws
+...
+Cloning into 'terraform-provider-aws'...
```

Comment:
I don't know if I'd add any of the output to these examples - it might be clearer to show just the commands that people will need to run

Reply:
Yeah, I was torn. I cleaned them up.